src/Drupal/Commands/core/LocaleCommands.php

```diff
@@ -128,4 +131,83 @@ public function update($options = ['langcodes' => self::REQ])
 
         drush_backend_batch_process();
     }
+
+    /**
+     * Exports to a gettext translation file.
+     *
+     * @command locale:export
+     *
+     * @param string $langcode The language code of the exported translations.
+     * @option $include
+     * @usage drush locale:export nl > nl.po
+     *   Export the Dutch non-customized translations.
+     * @usage drush locale:export nl --include=customized > nl.po
+     *   Export the Dutch customized translations.
+     * @usage drush locale:export > drupal.pot
+     *   Export the template file to translate.
+     * @see \Drupal\locale\Form\ExportForm::submitForm
+     *
+     * @throws \Exception
+     * @return string
+     *
+     * @aliases locale-export
+     * @drupal-dependencies locale
+     */
+    public function export($langcode = null, $options = ['include' => ['customized']])
+    {
+        // If template is required, language code is not given.
```

Comment:
Not clear why this code exists. Use locale_translatable_language_list() to check for valid languages. Use a (reusable) method for language validation.